python/cudf/cudf/core/dataframe.py

```diff
@@ -2066,6 +2066,10 @@ def iteritems(self):
     def loc(self):
         """
         Selecting rows and columns by label or boolean mask.
+        In case result is expected to be a single
```

Comment:
See below comment for `iloc`

Reply:
May be I can mention the case of pandas results in a Series in case of mixed dtypes and cudf has a limtation on that.

Reply:
Yeah - I think that would be more accurate